get-started/part3.md

```diff
@@ -146,10 +146,41 @@ Now let's run it. You have to give your app a name. Here, it is set to
 docker stack deploy -c docker-compose.yml getstartedlab
 ```
 
-See a list of the 5 containers you just launched:
+Our single service stack is running 5 container instances of our deployed image
+on one host. Let's investigate.
+
+Get the service ID for the one service in our application:
+
+```shell
+docker service ls
+```
+
+Swarm for Docker runs tasks that spawn containers. Tasks have state and their
+own IDs:
+
+```shell
+docker service ps <service>
+```
+
+>**Note**: SwarmKit tasks do not need to be containers (you can redefine them),
```

Comment:
Since SwarmKit hasn't been mentioned in this document before, should there be context about it? Something like "Docker Swarm is built using a project called SwarmKit", followed by the above.